RunningTheProduct.md

```diff
@@ -1,45 +1,44 @@
 ### Running the Product
 
+# Run the program by running the RedistricitingDriver.java file.

```

Comment:
There is no file named `RedistricitingDriver.java`  (This suggests that proofreading in general is needed.)